hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -209,26 +209,22 @@ private void handlePlatformTransaction(
 
             // Check all signature verifications. This will also wait, if validation is still ongoing.
             final var timeout = hederaConfig.workflowVerificationTimeoutMS();
-            final var maxMillis = instantSource.millis() + timeout;
             final var payerKeyVerification =
                     preHandleResult.verificationResults().get(preHandleResult.payerKey());
             if (payerKeyVerification.get(timeout, TimeUnit.MILLISECONDS).failed()) {
                 throw new HandleException(ResponseCodeEnum.INVALID_SIGNATURE);
             }
```

Comment:
The Payer Key should probably be verified by the signature verifier as well...

Reply:
Agree, let's move the fix from line 218 up to 212 and use the verifier to verify the payer key too

Reply:
done